styles/Home.module.css

```diff
@@ -2,6 +2,12 @@
   padding: 0 2rem;
 }
 
+.containerTitulo {
```

Comment:
In case we do need to use styles, it is not clear at all what is this style for.